miden-tx/src/tests/kernel_tests/test_account.rs

```diff
@@ -90,6 +90,9 @@ pub fn test_set_code_succeeds() {
             exec.account::incr_nonce
 
             exec.epilogue::finalize_transaction
+
+            # clean the stack
+            dropw dropw dropw dropw
```

Comment:
No, the initial idea was to move the stack truncation out if `finalize_transaction`, so we should handle it after this proc is executed. Strictly speaking we should truncate the stack properly using `movupw.3 dropw movupw.3 dropw movup.9 drop` set of instructions, but in this test we don't care about the elements left after `finalize_transaction`, so I decided to just drop entire stack since it is cheaper.